src/elements/content-sidebar/activity-feed/comment/components/__tests__/BaseCommentMenu.test.js

```diff
@@ -0,0 +1,100 @@
+// @flow
+
+import React from 'react';
+import { IntlProvider } from 'react-intl';
+import { fireEvent, render, type RenderResult } from '@testing-library/react';
+
+import { BaseCommentMenu } from '../BaseCommentMenu';
+import localize from '../../../../../../../test/support/i18n';
+
+import messages from '../../messages';
+
+jest.mock('react-intl', () => ({
+    ...jest.requireActual('react-intl'),
+    FormattedMessage: ({ defaultMessage }: { defaultMessage: string }) => <span>{defaultMessage}</span>,
+}));
+
+export const baseCommmentMenuDefaultProps: BaseCommentMenuProps = {
+    canDelete: true,
+    canEdit: true,
+    canResolve: true,
+    handleDeleteCancel: jest.fn(),
+    handleDeleteClick: jest.fn(),
+    handleDeleteConfirm: jest.fn(),
+    handleEditClick: jest.fn(),
+    handleMenuClose: jest.fn(),
+    handleStatusUpdate: jest.fn(),
+    isConfirmingDelete: false,
+    isResolved: false,
+    onSelect: jest.fn(),
+};
+
+const openMenu = (wrapper: RenderResult) => {
+    fireEvent.click(wrapper.queryByTestId('comment-actions-menu'));
+};
+
+// eslint-disable-next-line import/prefer-default-export
+export const getWrapper = props =>
```

Comment:
Why are we exporting from a unit test?

Reply:
Ah, it was just for dev. I was recycling these temporarily. The exports and unused defaultProps have since been removed.